lxd/util/http.go

```diff
@@ -132,20 +132,20 @@ type ContextAwareRequest interface {
 // CheckTrustState checks whether the given client certificate is trusted
 // (i.e. it has a valid time span and it belongs to the given list of trusted
 // certificates).
-func CheckTrustState(cert x509.Certificate, trustedCerts []x509.Certificate) bool {
+func CheckTrustState(cert x509.Certificate, trustedCerts map[string]x509.Certificate) (bool, string) {
```

Comment:
Please update the docstring to mention that the function will also return the key of the given map of certs that corresponds to a trusted cert.